src/app/data-planes/locales/en-us/index.yaml

```diff
@@ -17,9 +17,15 @@ data-planes:
         data-plane-clusters-view: 'Clusters'
         data-plane-config-view: 'YAML'
       overview: 'Overview'
+      gateway: 'Gateway'
+      inbounds: 'Inbounds'
+      inbound_name: 'Service: {service}'
```

Comment:
**Question**: Should the inbound heading/title be prefixed with "Service:" or not?

See @johncowen’s question about that here: https://github.com/kumahq/kuma-gui/pull/1745#pullrequestreview-1731817036

Reply:
Somewhat related: I added the copy button to this title now.

Reply:
I’ve removed the "Service:" prefix for now. Let’s see if people find it confusing.